templates/cluster/gcp-standalone-cp/values.yaml

```diff
@@ -0,0 +1,74 @@
+  name: "" # @schema description: The GCP Service Account credentials secret name, auto-populated; type: string
+
+clusterNetwork: # @schema description: The cluster network configuration; type: object
+  apiServerPort: 6443 # @schema description: The port the API Server should bind to; type: number; minimum: 1; maximum: 65535
+  pods: # @schema description: The network ranges from which Pod networks are allocated; type: object
+    cidrBlocks: # @schema description: A list of CIDR blocks; type: array; item: string
+      - "10.244.0.0/16"
+  services: # @schema description: The network ranges from which service VIPs are allocated; type: object
+    cidrBlocks: # @schema description: A list of CIDR blocks; type: array; item: string
+      - "10.96.0.0/12"
+
+clusterLabels: {} # @schema description: Labels to apply to the cluster; type: object; additionalProperties: true
+clusterAnnotations: {} # @schema description: Annotations to apply to the cluster; type: object; additionalProperties: true
+
+# GCP cluster parameters
+project: "" # @schema description: The name of the project to deploy the cluster to; type: string; required: true
+region: "" # @schema description: The GCP Region the cluster lives in; type: string; required: true
+network: # @schema description: The GCP network configuration; type: object
+  name: "default" # @schema description: The GCP network name; type: string; required: true
+  mtu: 1460 # @schema description: Maximum Transmission Unit in bytes; type: number; minimum: 1300; maximum: 8896
+additionalLabels: {} # @schema description: Additional set of labels to add to all the GCP resources; type: object; additionalProperties: true
+
+# GCP machines parameters
+controlPlane: # @schema description: Control plane parameters; type: object
+  instanceType: "" # @schema description: The type of instance to create. Example: n1.standard-2; type: string; required: true
+  subnet: "" # @schema description: A reference to the subnetwork to use for this instance. If not specified, the first subnetwork retrieved from the Cluster Region and Network is picked; type: string; patternProperties: {"^[a-z]$": {"type": "string"}}
+  providerID: "" # @schema description: The unique identifier as specified by the cloud provider; type: string
+  imageFamily: "" # @schema description: The full reference to a valid image family to be used for this machine; type: string
+  image: "" # @schema description: The full reference to a valid image to be used for this machine. Takes precedence over imageFamily; type: string
+  additionalLabels: {} # @schema description: Additional set of labels to add to an instance, in addition to the ones added by default by the GCP provider. If both the GCPCluster and the GCPMachine specify the same tag name with different values, the GCPMachine's value takes precedence; type: object; additionalProperties: true; patternProperties: {"^[a-z]$": {"type": "string"}}
```

Comment:
the same question regarding pattern: why is it not allowed to have capitals or digits? e.g. `K0rdent`. the `.Values.additionalLabels` field does not have the same pattern

I believe there is the same question regarding all of the `patternProperties`

Reply:
You're right. The pattern should be adapted everywhere.